fiscal_epos_print/static/src/js/epson_epos_print.js

```diff
@@ -64,11 +65,13 @@ export class EpsonEposPrint {
                 info = addInfo[tagStatus[0]];
                 msgPrinter = this.decodeFpStatus(info);
             }
+            let order_lines_msg = order.orderlines.map((l) => `\n${l.full_product_name}, Quantity: ${l.quantity}, Price: ${l.price}, Discount: ${l.discount}`).join(",");
```

Comment:
E se `order` è `""`?

Reply:
In teoria non può essere vuoto ne il carrello ne il riepilogo con i pagamenti per poter effettuare uno scontrino
per questo non lo avevo messo il controllo
Metto lo stesso?